com.avaloq.tools.ddk.check.core.test/src/com/avaloq/tools/ddk/check/core/generator/IssueCodeValueTest.xtend

```diff
@@ -64,11 +64,21 @@ class IssueCodeValueTest extends AbstractCheckGenerationTestCase {
             }
           }
         }
+
+        live error MYCheck3 "Label 3"
+        message "Message 3" {
+          for Documented elem {
+            switch elem {
+              Context : issue on elem
+              Check : issue on elem
+            }
+          }
+        }
       }
     ''';
     // @Format-On
 
-    val expectedIssueCodeValues = #{'MY_CHECK_1' -> 'MyCheck1', 'MY_CHECK_2' -> 'MyCheck2'}
+    val expectedIssueCodeValues = #{'MY_CHECK_1' -> 'MyCheck1', 'MY_CHECK_2' -> 'MyCheck2', 'MY_CHECK_3' -> 'MyCheck3'}

```

Comment:
Should the expected not be "MyCheck3"?

Reply:
it is. 'MY_CHECK_3' -> 'MyCheck3' sets the expectation that there will be an issue code "MY_CHECK_3" that has a value ending in "MyCheck3"

Reply:
Sorry, I meant: Should the expected not be "MYCheck3"?, with MY in two capital letters